src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-console.log('hello mate academy!');
+const galleryList = document.querySelector('.gallery__list');
+const bigImage = document.querySelector('.gallery__large-img');
+
+function changeImage(evt) {
+  evt.preventDefault();
+
+  const item = evt.target.closest('.list-item__link');
+  const imgValue = item.getAttribute('href');
```

Comment:
ты ведь берешь аттрибут href, называй переменную соответствующе
```suggestion
  const href = item.getAttribute('href');
```